src/formatter/progress_bar_formatter_spec.ts

```diff
@@ -94,7 +94,7 @@ describe('ProgressBarFormatter', () => {
         },
         {
           data:
-            'Feature: a\nScenario: b\nGiven a step\nWhen a step\nThen a step',
+            'Feature: a\nRule: b\nExample: c\nGiven a step\nWhen a step\nThen a step',
```

Comment:
Can we please create an additional test for this? (and retain the old one)

Reply:
Done